specification/blueprint/resource-manager/readme.md

```diff
@@ -0,0 +1,154 @@
+# Blueprint
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Blueprint.
+
+---
+## Getting Started
+To build the SDK for Blueprint, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+### Basic Information
+These are the global settings for the Blueprint API.
+
+``` yaml
+title: BlueprintManagementClient
+description: Blueprint Client
+openapi-type: arm
+tag: package-2017-11-preview
+```
+
+
+### Tag: package-2017-11-preview
+
+These settings apply only when `--tag=package-2017-11-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-11-preview'
+input-file:
+- Microsoft.Blueprint/preview/2017-11-11-preview/Blueprint.json
```

Comment:
Path to spec file is case sensitive, so this should be `Microsoft.Blueprint/preview/2017-11-11-preview/blueprint.json`